AcadeMeData/migrations/0001_initial.py

```diff
@@ -1,5 +1,3 @@
-# Generated by Django 3.2.9 on 2021-12-25 15:57
-
 import django.core.validators
```

Comment:
Please remove this file from PR. 
It should stay unchanged

Reply:
done